src/DotLiquid.Tests/ContextTests.cs

```diff
@@ -233,6 +233,24 @@ public void TestVariables()
             Assert.That(_context["nil"], Is.EqualTo(null));
         }
 
+        [Test]
+        public void TestVariablesArray()
+        {
+            List<int> list = new List<int> { 1, 2, 3, 4, 5 };
+            _context["list"] = list;
+            Assert.That(_context["list"], Is.EqualTo(list));
+            Assert.That(_context["list[0]"], Is.EqualTo(list[0]));
+            Assert.That(_context["list[-1]"], Is.EqualTo(list[list.Count - 1]));

```

Comment:
```suggestion
            Assert.That(_context["list[0]"], Is.EqualTo(1));
            Assert.That(_context["list[-1]"], Is.EqualTo(5));
```
expectations should be as concrete as possible.